tensorflow/core/common_runtime/gpu/gpu_cudamallocasync_allocator.cc

```diff
@@ -386,7 +386,9 @@ bool GpuCudaMallocAsyncAllocator::ClearStats() {
 
 void GpuCudaMallocAsyncAllocator::SetStreamAndPreallocateMemory(void* stream) {
 #if TF_CUDA_MALLOC_ASYNC_SUPPORTED
-  if (cuda_stream_ != nullptr) {
+  CUstream new_cuda_stream = *(reinterpret_cast<CUstream*>(stream));
```

Comment:
Prefer `static_cast`, it is well-defined here.

Reply:
Fixed, thanks